{"name": "test_data[2]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 2 == 3", "trace": "new_data = 2\n\n    def test_data(new_data):\n        print(\"测试步骤\")\n>       assert new_data == 3\nE       assert 2 == 3\n\nscripts\\test_fixture_params.py:17: AssertionError"}, "parameters": [{"name": "new_data", "value": "2"}], "start": 1578278406209, "stop": 1578278406209, "uuid": "496b3358-42e4-42e9-8406-cfabfa7c0cc6", "historyId": "70c9b1e55e3198f772f3c0f14daa97e5", "testCaseId": "dd465c263251d58de65400669ad5358e", "fullName": "scripts.test_fixture_params#test_data", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test_fixture_params"}, {"name": "host", "value": "DESKTOP-TJSPRKG"}, {"name": "thread", "value": "11684-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test_fixture_params"}]}